### PR TITLE
fix adhoc pipelines and centos 6 omnibus test

### PR DESCRIPTION
**Pull request by evanahlberg**
Signed-off-by: Evan Ahlberg <evanahlberg@gmail.com>

## Description

Make the pre-command run on the new validate adhoc/release pipelines.
Revert change to omnibus-test.sh to fix failing centos 6 test changed in this PR:
https://github.com/chef/chef/pull/13489

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: .buildkite/hooks/pre-command [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -eu
 
 # Only execute in the verify pipeline
-[[ "$BUILDKITE_PIPELINE_NAME" =~ verify$ ]] || exit 0
+[[ "$BUILDKITE_PIPELINE_NAME" =~ verify$ ]] || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/release$ ]] || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/adhoc$ ]]  || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/adhoc-canary$ ]] || exit 0
```
Comment (tpowell-progress) [inline .buildkite/hooks/pre-command:6]: shouldn't you be able to just match on leading `'validate/'`

Comment (evanahlberg) [reply .buildkite/hooks/pre-command]: yeah no problem, updated!

**Review by tpowell-progress — APPROVED** [review]

File: omnibus/omnibus-test.sh [hunk 2]
```diff
@@ -120,16 +120,14 @@ export CHEF_LICENSE=accept-no-persist
...
   sudo -E bundle install --jobs=3 --retry=3
   sudo -E bundle exec rspec --profile -f progress
-else
-  sudo bundle install --jobs=3 --retry=3
-  sudo bundle exec rspec --profile -f progress
-fi
+# else
+#   sudo bundle install --jobs=3 --retry=3
+#   sudo bundle exec rspec --profile -f progress
+# fi
```
Comment (jaymzh) [inline omnibus/omnibus-test.sh:133]: You said you're reverting a PR - can you link to it in the PR description?

Also, does this FIX C6? If so, lets just delete this code, not comment it out.

If not, why are we dropping C6 code?

**Review by jaymzh — CHANGES_REQUESTED** [review]

File: omnibus/omnibus-test.sh [hunk 3]
```diff
@@ -120,16 +120,14 @@ export CHEF_LICENSE=accept-no-persist
...
 cd "$chef_gem"
 
-# only add -E if not on centos 6
-sudo_path="$(command -v sudo)"
-# cspell:disable-next-line
-rhel_sudo="/opt/rh/devtoolset-7/root/usr/bin/sudo"
-sudo_args=""
-if [[ "$sudo_path" != "$rhel_sudo" ]]; then
-  echo "HERE"
+# # only add -E if not on centos 6
```
[the remark this answers is not in the document]
Comment (evanahlberg) [reply omnibus/omnibus-test.sh]: done!

File: omnibus/omnibus-test.sh (hunk 2 above)
Comment (evanahlberg) [reply omnibus/omnibus-test.sh]: Added the PR to descripton.

The code was to fix C6 running build/test inside a container, I removed it for now and will re-add when we are ready to use the container.

**Review by tpowell-progress — APPROVED** [review]

**Comment by tpowell-progress** [comment]
Bypassed protections due to concerns being addressed.

**Pull request merged**